app/lib/efile/dependent_eligibility/qualifying_child.rb

```diff
@@ -41,15 +41,17 @@ def qualified_student?
       end
 
       def over_qualifying_age_limit?
-        age > 18
-      end
+        return age >= 17 if tax_year == 2020
 
-      private
+        age >= 18
```

Comment:
hmm... this isnt quite right. The qualifyng age limit for qualifying children is being less than 19.

The rules for CTC are more specific -- only a subset of qualifying children are qualified for getting the CTC.

I think we need a separate rules file for the CTC that defines a more strict rule for age instead of overwriting the qualifying child age rules.

Reply:
you could also create _another_ method thats like age_qualifies_for_ctc? that defines this logic

Reply:
It might seem like overkill, but I was imagining classes like

Efile::DependentEligibility::Ctc for each benefit

where self.rules looks like:

```
def self.rules
{ qualifying_child_test: :qualifying_child?,
  age_test: :under_ctc_age_test?,
  tin_test: :ssn_or_atin?
}

def qualifying_payment
  return 0 unless qualifies?
  $$$ if age_during(tax_year) < 6
  $$$ if age_during(tax_year) > 6
end
```